experimental/generation/generator/templates/library-operations.en-us.data

```diff
@@ -1,14 +1,23 @@
 > Add only
 - add {@AddToProperty=${entity}}
+- with {@AddToProperty=${entity}}
 - add {@AddToProperty=${entity} and ${entity}}
+- with {@AddToProperty=${entity} and ${entity}}
 - add {@AddToProperty=${entity}, ${entity} and ${entity}}
+- with {@AddToProperty=${entity}, ${entity} and ${entity}}
```

Comment:
Could we set 4 or more entities at a time ?

Reply:
We can improve this in the future.